src/quo2/components/list_items/account/view.cljs

```diff
@@ -0,0 +1,138 @@
+         [selectable?])
+        [rn/pressable
+         {:style               (style/container
+                                {:state @state :blur? blur? :customization-color customization-color})
+          :on-press-in         on-press-in
+          :on-press-out        on-press-out
+          :accessibility-label :container}
+         [rn/view {:style style/left-container}
+          [account-avatar/view (assoc account-props :size 32)]
+          [rn/view {:style style/account-container}
+           [rn/view
+            {:style style/account-title-container}
+            [text/text
+             {:weight :semi-bold
+              :size   :paragraph-1}
+             (:name account-props)]
+            (when title-icon?
+              [rn/view
+               {:style               style/keycard-icon-container
+                :accessibility-label :keycard-icon}
+               [icon/icon :i/keycard
+                {:size  20
+                 :color (if blur?
+                          colors/white-opa-40
+                          (colors/theme-colors colors/neutral-50 colors/neutral-40 theme))}]])]
+           [text/text {:size :paragraph-2}
+            [text/text
+             {:size   :paragraph-2
+              :weight :monospace
+              :style  (style/account-address blur? theme)}
+             (:address account-props)]]]]
+         [rn/view {:style (when (= type :tag) style/token-tag-container)}
+          (when (or (= type :balance-neutral)
+                    (= type :balance-negative)
+                    (= type :balance-positive))
+            [rn/view
+             {:style               style/balance-container
+              :accessibility-label :balance-container}
+             [text/text
+              {:weight :medium
+               :size   :paragraph-2}
+              (:fiat-value balance-props)]
+             [rn/view
+              {:style style/metrics-container}
+              [text/text
+               {:size  :paragraph-2
+                :style (style/metric-text type theme)}
+               (str (:percentage-change balance-props) "%")]
+              [rn/view {:style (style/dot-divider type theme)}]
+              [text/text
+               {:size  :paragraph-2
+                :style (style/metric-text type theme)}
+               (:fiat-change balance-props)]
+              (when (not= type :balance-neutral)
+                [rn/view
+                 {:style               style/arrow-icon-container
+                  :accessibility-label :arrow-icon}
+                 [icon/icon (if (= type :balance-positive) :i/positive :i/negative)
+                  (assoc (style/arrow-icon type theme)
+                         :accessibility-label
+                         (if (= type :balance-positive) :icon-positive :icon-negative))]])]])
+          (when (= type :tag)
+            ;; TODO: Use Tiny tag component when available (issue #17341)
```

Comment:
Would be good to mention this PR and this TODO in the `tiny tag` issue so it doesn't get lost.

Reply:
Done!